docs-v2/integrations/all/brightcrowd.mdx

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
If you can follow other documentation models that would be great. No need to fill information if you are not sure